Rakefile

```diff
@@ -1,49 +1,51 @@
 # frozen_string_literal: true
 
-require 'rake/testtask'
-require 'rubocop/rake_task'
-require 'bundler/gem_tasks'
-require 'rspec/core/rake_task'
+require "rake/testtask"
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
 
 RSpec::Core::RakeTask.new(:spec)
 
 # run tests
-desc 'default checks'
-task default: %i{test:unit}
+desc "default checks"
+task default: %i(test:unit)
 
 # Rubocop
-desc 'Run Rubocop lint checks'
+desc "Run Rubocop lint checks"
 task :style do
   RuboCop::RakeTask.new
 end
 
 # lint the project
-desc 'Run robocop linter'
-task lint: [:rubocop]
+require "chefstyle"
+require "rubocop/rake_task"
+RuboCop::RakeTask.new(:lint) do |task|
+  task.options << "--display-cop-names"
+end
 
 # Automatically generate a changelog for this project. Only loaded if
 # the necessary gem is installed.
 # use `rake changelog to=1.2.0`
 begin
-  v = ENV['to']
-  require 'github_changelog_generator/task'
+  v = ENV["to"]
+  require "github_changelog_generator/task"
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
     config.future_release = v
   end
 rescue LoadError
-  puts '>>>>> GitHub Changelog Generator not loaded, omitting tasks'
+  puts ">>>>> GitHub Changelog Generator not loaded, omitting tasks"
 end
```

Comment:
Do we know if we need this GitHubChangelogGenerator? If yes, we may need to include the gem `github_changelog_generator` in the Gemfile inorder to avoid the error which we are facing currently i.e. 
`>>>>> GitHub Changelog Generator not loaded, omitting tasks`